python/bigipconfigdriver.py

```diff
@@ -424,13 +424,14 @@ def create_ltm_config_kubernetes(bigip, config):
         new_pool['loadBalancingMode'] = balance
         new_pool['partition'] = pool['partition']
         if pool['name'] in f5_services or vname in f5_services:
-            found_svc = True
-            for member in pool['poolMemberAddrs']:
-                members.update({member: {
-                    'state': 'user-up',
-                    'session': 'user-enabled'
-                }})
-        new_pool['members'] = members
+            if pool['poolMemberAddrs'] is not None:
+                found_svc = True
+                for member in pool['poolMemberAddrs']:
+                    members.update({member: {
+                        'state': 'user-up',
+                        'session': 'user-enabled'
+                    }})
+            new_pool['members'] = members
         configuration['pools'].append(new_pool)
         if not found_svc:
             log.warning(
```

Comment:
I don't think this is an exceptional case any more and a warning is too sensitive as it indicates something went wrong. Let's move this to info.